esp-hal/src/lib.rs

```diff
@@ -156,28 +154,28 @@ pub use xtensa_lx_rt::{self, entry};
 #[cfg(any(esp32, esp32s3))]
 pub use self::soc::cpu_control;
 #[cfg(efuse)]
+#[instability::unstable]
+#[cfg_attr(not(feature = "unstable"), allow(unused))]
```

Comment:
This is probably something that can be fixed in instability.

Reply:
Would you mind filing an issue so we can come back to this?

Reply:
Let's fix this separately once the PR is merged :)